cli/flox/src/commands/include.rs

```diff
@@ -17,7 +17,7 @@ pub enum IncludeCommands {
     #[bpaf(command, hide)]
     Help,
     /// Upgrade an environment with latest changes to its included environments
-    #[bpaf(command, header(indoc! {"
+    #[bpaf(command, long("update"), header(indoc! {"
```

Comment:
Were you thinking that this would close out https://github.com/flox/flox/issues/3796? I'm not sure that it solves it yet because the alias isn't discoverable or documented.

Reply:
I was thinking of this more as "if someone accidentally types update, they get what they want." I don't think we need it to be super discoverable beyond that?